test/docgen/view/__snapshots__/enum.test.ts.snap

```diff
@@ -32,23 +32,25 @@ Object {
 exports[`csharp snapshot 2`] = `
 " TagMutability <a name=\\"TagMutability\\" id=\\"@aws-cdk/aws-ecr.TagMutability\\"></a>
 
+The tag mutability setting for your repository.
+
+# Enum Members <a name=\\"Enum Members\\" id=\\"Enum Members\\"></a>
```

Comment:
Because of the way a `MarkdownDocument`'s "sections" are always rendered after their "lines", I noticed example snippets would get rendered after the table listing all of the available enums. The cleanest fix I saw for this was to put the enum members + table in its own section, similar to what we do for classes/structs/interfaces/etc. - though it did cause some extra changes in the snapshots.